Kernel/Time/TimeManagement.cpp

```diff
@@ -77,7 +77,7 @@ ErrorOr<void> TimeManagement::validate_clock_id(clockid_t clock_id)
     };
 }
 
-Time TimeManagement::current_time(clockid_t clock_id) const
+Duration TimeManagement::current_time(clockid_t clock_id) const
```

Comment:
i feel it should return this `UnixDateTime` now… 🫤

Reply:
No, because different clocks should technically return different time types. This change will not be addressed in this PR since it is a little more complicated.